articles/Image Search: Leveraging Image Models, Large Language Models, and Multimodal Models/README.md

```diff
@@ -0,0 +1,169 @@
+
+![figure 11 mm-llms.png](figure%2011%20mm-llms.png#center)  
+ <p><center>Fig 11. MM-LLMs<sup>[14]</sup>. </center></p> 
+
+MM-LLMs focusing on multimodal understanding typically encompass just the first three components: modality encoders, the core LLM backbone, and modality generators. Throughout the training phase, these elements are generally maintained in a frozen state. Optimization efforts are concentrated on the input and output projectors, which are relatively lightweight. As a result, a small fraction of the overall parameters—commonly about 2%—are actually trainable within MM-LLMs. This percentage is determined by the size of the principal LLM integrated into the MM-LLM framework. Due to this configuration, MM-LLMs can undergo cost-effective training, making the enhancement of performance in assorted multimodal tasks more attainable.  
+
+![figure 12 The general model architecture of MM-LLMs and the implementation choices for each component.png](figure%2012%20The%20general%20model%20architecture%20of%20MM-LLMs%20and%20the%20implementation%20choices%20for%20each%20component.png#center)    
+<center>Fig12. The general model architecture of MM-LLMs and the implementation choices for each component<sup>[14]</sup>.  </center>  </p>  
+
+In this undertaking, we carried out validations and tests for image caption generation using GPT4 vision, LLaVa<sup>[15]</sup><sup>[16]</sup>, and Qwen-vl<sup>[13]</sup>models. We meticulously analyzed the precision of the image descriptions provided by these models. Given that GPT4 vision has not been made available for open source use, we based our further enhancements on the Qwen-vl model, selecting it as our foundational model for fine-tuning after our comparative assessments were concluded.  
+## 4. Conclusion  
+During our image retrieval endeavor, we have diligently addressed the distinctive traits of images pertinent to the business sector, taking into account crucial image features, textual information, and details from both sides of the image. By analyzing and valorizing this information, we have implemented strategies such as reverse image search, textual retrieval, and image search via text. Our deployment of advanced deep learning algorithms encompasses an array of models—ranging from those specialized in computer vision and natural language processing to multimodal algorithms, large language models, and comprehensive multimodal language-vision models. Through systematic enhancement of the performance across each model phase, we have successfully realized a retrieval process that is both effective and precise for images and accompanying text.  
+## 5. Reference
```

Comment:
You do not need ##5. Reference usually is not numbered.

Reply:
Already modified